### PR TITLE
[WIP] Update resources for ASF landing

### PR DESCRIPTION
**Pull request by jbonofre**
This PR prepares the resources to land at ASF.

### CODE REVIEW

File: .asf.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+github:
+  description: "Apache Polaris"
+  homepage: https://polaris.apache.org/
+  labels:
+    - polaris
+    - iceberg
+    - apache
+
+  enabled_merge_buttons:
+    merge: false
```
Comment (snazy) [inline .asf.yml:32]: Should we enable `merge` temporarily to have "one big bang PR" for "all the things"?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,61 +1,56 @@
...
+
+When filing an [issue](https://github.com/apache/polaris/issues), make sure to answer these five questions:
+1. What version of Apache Polaris are you using?
 2. What operating system and processor architecture are you using?
 3. What did you do?
 4. What did you expect to see?
 5. What did you see instead?
 
-Troubleshooting questions should be posted in [GitHub Discussions](https://github.com/polaris-catalog/polaris/discussions/categories/q-a) instead of the issue tracker. Maintainers and community members will answer your questions there or ask you to file an issue if you’ve encountered a bug. 
+Troubleshooting questions should be posted on [dev mailing list](https://lists.apache.org/list.html?dev@polaris.apache.org) instead of the issue tracker. Maintainers and community members will answer your questions there or ask you to file an issue if you’ve encountered a bug.
```
Comment (snazy) [inline CONTRIBUTING.md:41]: Add the subscribe link?

File: .asf.yml (hunk 1 above)
Comment (jbonofre) [reply .asf.yml:32]: Good point. Let me enable `merge` (at least temp). Thanks !

File: CONTRIBUTING.md (hunk 2 above)
Comment (jbonofre) [reply CONTRIBUTING.md:41]: It makes sense.

**Comment by jbonofre** [comment]
Forgot to add `DISCLAIMER_WIP`, let me do it.

**Comment by jbonofre** [comment]
Superseeded by https://github.com/apache/polaris/pull/174

**Pull request closed without merging**